mobile/EmbeddedCounselDemo/EmbeddedCounselDemo/AccessCodeView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  AccessCodeView.swift
+//  EmbeddedCounselDemo
+//
+//  Created by Counsel on 5/12/25.
```

Comment:
nit: stick with Created by Ray Kim or Created by Counsel. I'm good with either

Reply:
Yep, they should all be Counsel.